ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextViewManager.java

```diff
@@ -80,6 +81,22 @@ public void setLineHeight(ReactTextView view, float lineHeight) {
     }
   }
 
+  @ReactProp(name = ViewProps.TEXT_DECORATION_LINE)
+  public void setTextDecorationLine(ReactTextView view, @Nullable String textDecorationLine) {
+    int paintFlags = view.getPaintFlags();
+    if (textDecorationLine == null || "none".equals(textDecorationLine)) {
+      view.setPaintFlags(paintFlags & ~Paint.UNDERLINE_TEXT_FLAG & ~Paint.STRIKE_THRU_TEXT_FLAG);
+    } else if ("underline".equals(textDecorationLine)) {
+      view.setPaintFlags(paintFlags | Paint.UNDERLINE_TEXT_FLAG & ~Paint.STRIKE_THRU_TEXT_FLAG);
+    } else if ("line-through".equals(textDecorationLine)) {
+      view.setPaintFlags(paintFlags & ~Paint.UNDERLINE_TEXT_FLAG | Paint.STRIKE_THRU_TEXT_FLAG);
+    } else if ("underline line-through".equals(textDecorationLine)) {
```

Comment:
Good point about x-platform parity. Would you like to submit a follow-up PR that adds flexible parsing to Android and iOS? That'd be awesome!